app/src/renderer/components/staking/PageStaking.vue

```diff
@@ -1,7 +1,7 @@
 <template lang="pug">
 tm-page(title='Staking')
   div(slot="menu"): tool-bar
-    a(@click='connected && updateDelegates()' v-tooltip.bottom="'Refresh'" :disabled="!connected")
```

Comment:
bad resolve on a merge conflict?

Reply:
what should it be? it looks like your disabled PR work @fabo but i don't see the error.

Reply:
fixed @faboweb